sdk/core/Azure.Core.Amqp/src/AmqpMessageBody.cs

```diff
@@ -35,6 +45,30 @@ public AmqpMessageBody(IEnumerable<ReadOnlyMemory<byte>> data)
             BodyType = AmqpMessageBodyType.Data;
         }
 
+        /// <summary>
+        /// Initializes a new <see cref="AmqpMessageBody"/> instance with the
```

Comment:
Do we expect callers using this type to have protocol knowledge for AMQP?   If so, maybe we should consider explicitly calling out what part of the message each constructor uses.  That way, it's clear what the shape of the message is for those who may be trying to align to a certain one for interop with other systems.

Reply:
Added links to the AMQP docs.